### ISSUE TITLE
Support Android Nougat apks

### ISSUE THREAD

**Issue by Blystad** [skylot/jadx#144]
Hey,

Nougat added [dex version 037](https://source.android.com/devices/tech/dalvik/dex-format.html), which it seems JADX fails to decompile. I get:

```
13:44:31 INFO  - output directory: classes
13:44:31 ERROR - jadx error: Unexpected magic: [100, 101, 120, 10, 48, 51, 55, 0]
com.android.dex.DexException: Unexpected magic: [100, 101, 120, 10, 48, 51, 55, 0]
	at com.android.dex.TableOfContents.readHeader(TableOfContents.java:79) ~[dx-1.10.jar:na]
	at com.android.dex.TableOfContents.readFrom(TableOfContents.java:69) ~[dx-1.10.jar:na]
```

My temporary hack was to just change 37 to 35, Then jadx decompiled it (albeit with some errors).

**Comment by cyroxx** [comment]
This might be solved by merging pull request #114.

**Comment by harshleen8** [comment]
I'm getting an error, "com.android.build.api.transform.TransformException.com.android.dex.DexException.Unexpected magic:[0*00,0*00,0*00,0*00]"
what shall I do, to remove this error.??
